keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -330,6 +343,11 @@ There are already e2e tests running that verify correctness of this feature - ht
   -  There are already e2e tests running that verify correctness of this feature -
   https://testgrid.k8s.io/presubmits-kubernetes-nonblocking#pull-kubernetes-e2e-gce-cos-alpha-features
   
+### GA  
+
+- The feature has been extensively tested in CI and deployed with drivers in real world. For example - https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml#L166
+- The test grid already has tests for the feature.
```

Comment:
Do we have both presubmits and periodics executing this feature?

Reply:
The tests are still running only in alpha jobs, despite the feature is already in beta. I am going to LGTM the KEP, but I will block feature gate graduation until the test run in regular e2e jobs.

Reply:
So, we do not have ability to run full lifecycle of recovery with normal driver. It is hard to setup cases where expansion will fail for various scenarios with say EBS or GCE-PD driver. To setup custom recovery cases, we unfortunately need hostpath/mock driver. There is no easy way to exercise all possible scenarios otherwise.

With https://github.com/kubernetes/kubernetes/pull/131593/files we now verify expansion related fields and whether they are correctly setup. I had another WIP PR, which checks inflight stuff, which will racy for online expansion but will work for offline at least. I will push it and make it a proper PR.